test/suite_test.go

```diff
@@ -49,6 +49,7 @@ var (
 	clusterType                  = os.Getenv("E2E_CLUSTER_TYPE")
 	envSkipBuildImages           = os.Getenv("E2E_SKIP_BUILD_IMAGES")
 	envTestImage                 = os.Getenv("E2E_SPO_IMAGE")
+	envSelinuxdTestImage         = os.Getenv("E2E_SELINUXD_IMAGE")
```

Comment:
Can this be generalized and upstreamed?

Reply:
I think we can also make it upstream in the future as we can choose what selinuxd image to use for testing